pl/lessons/basics/functions.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: page
+title: Funkcje
+category: basics
+order: 7
+lang: pl
+---
+
+W Elixirze tak, jak w wielu innych językach funkcyjnych, funkcje należą do bytów podstawowych (ang. _first class 
```

Comment:
Comma is in bad place. It's:
W Elixirze tak, jak w...
Should be:
W Elixirze, tak jak w...